test/fixtures/spawn_closed_stdio.py

```diff
@@ -0,0 +1,9 @@
+import os
+import sys
+import subprocess
+os.close(0)
+os.close(1)
+os.close(2)
+cmd = sys.argv[1] + ' -e "process.stdin; process.stdout; process.stderr;"'

```

Comment:
I think I would prefer it if the arguments came from the test file, i.e. you just spawn `sys.argv[1]` here and move the `-e … ` bits to `test-stdio-closed.js`